server/utils/s3Operations.js

```diff
@@ -0,0 +1,31 @@
+import s3 from 's3';
+
+let client = s3.createClient({
+	maxAsyncS3: 20, 
+	s3RetryCount: 3, 
+	s3RetryDelay: 1000,
+	multipartUploadThreshold: 20971520, // this is the default (20 MB) 
+	multipartUploadSize: 15728640, // this is the default (15 MB) 
+	s3Options: {
+		accessKeyId: `${process.env.AWS_ACCESS_KEY}`,
+		secretAccessKey: `${process.env.AWS_SECRET_KEY}`,
+	},
+});
+
+export function deleteObjectFromS3(url) {
+	let objectKey = url.split("/").pop();
+	let params = {
+		Bucket: `${process.env.S3_BUCKET}`,
+		Delete: {
+			Objects: [
+				{
+        			Key: objectKey,
+      			},
+			],
+		},
+	};
+	let del = client.deleteObjects(params);
+	del.on('end', function() {
```

Comment:
What you want to do here is add a second parameter to `deleteObjectFromS3`, that is a callback function, i.e. `export function deleteObjectFromS3(url, callback) { ...`. Then, here, you call that callback. 

For functions that use `deleteObjectFromS3`, you would use it like this:
```javascript
deleteObjectFromS3(url, function() {
  //code here that you want to get executed after file is deleted from S3
});
```